src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/fields/SingleSelection.test.js

```diff
@@ -1143,3 +1142,46 @@ test('Should throw an error if no "resource_key" option is passed in fieldOption
         />
     )).toThrowError(/"resource_key"/);
 });
+
```

Comment:
Can you please also add a test that adds options via the `requestOptions` and `dataPathToAutoComplete`? Want to make sure that the combination also works properly (including how it resolves conflicts when both write to the same parameter, not that this accidentally changes in the future).